### PR TITLE
Added Support for Thermal Foundation/Expansion Fuels

### PR DESCRIPTION
**Pull request by EweLoHD**
Added Support for "Naphtha" and "Refined Fuel". Are the values that I gave them good?

--> #302

### CODE REVIEW

**Comment by FriedrichLP** [comment]
that should be already possible since you can add custom diesel fuels in the config. Maybe you can add them to the default fuel map

File: src/main/java/cam72cam/immersiverailroading/Config.java [hunk 1]
```diff
@@ -19,6 +19,9 @@
 		@Comment({ "Enable Boiler Explosions" })
 		public static boolean explosionsEnabled = true;
 
+		@Comment({ "Enable environmental damage of Boiler Explosions"})
+		public static boolean explosionEnvDamageEnabled = true;
+
```
Comment (cam72cam) [inline src/main/java/cam72cam/immersiverailroading/Config.java]: Why is this in here?

File: src/main/java/cam72cam/immersiverailroading/Config.java [hunk 2]
```diff
@@ -101,7 +104,8 @@
 			Items.EMERALD.getRegistryName().toString()
 		};
 		
-		@Comment("Fuels for diesel Locomotives")
+		@Comment("Fuels for diesel Locomotives" +
```
Comment (cam72cam) [inline src/main/java/cam72cam/immersiverailroading/Config.java]: What does the \n look like in the in-game UI and in the config text?

Comment (EweLoHD) [reply src/main/java/cam72cam/immersiverailroading/Config.java]: image

File: src/main/java/cam72cam/immersiverailroading/Config.java (hunk 1 above)
Comment (EweLoHD) [reply src/main/java/cam72cam/immersiverailroading/Config.java]: idk, this is not new. Had some Problems with Github Desktop recently

File: src/main/java/cam72cam/immersiverailroading/Config.java (hunk 2 above)
Comment (EweLoHD) [reply src/main/java/cam72cam/immersiverailroading/Config.java]: and it doesn't show me comments in the in-game UI

**Pull request merged**